vars/main.yml

```diff
@@ -230,10 +230,10 @@ manala_apt_keys_patterns:
     id:  86E50310
   sury_php_debian:
     url: https://packages.sury.org/php/apt.gpg
-    id: 4A7A714D
+    id:  4A7A714D
   sury_php_ubuntu:
     keyserver: hkp://keyserver.ubuntu.com:80
-    id: E5267A6C
+    id:        E5267A6C
```

Comment:
Ok got it, strange approach though (to me it's either _all values_ are aligned with all others or not at all, not per entry indent).

But following your logic, I can still find some incorrect ones :) https://github.com/manala/ansible-role-apt/pull/15/files/edd094fd88a79a112b0d1c099ae5e7f9321e6e38..f1f6e33b0ee9cfb3ff27937aa6f8f217cabdc914#diff-6d62b73b3f46cfa11f6466d08cf45303R132